druid-shell/src/windows/mod.rs

```diff
@@ -537,16 +545,23 @@ impl WndProc for MyWndProc {
                     let s = s.as_mut().unwrap();
                     let key_code: KeyCode = (wparam as i32).into();
                     s.stashed_key_code = key_code;
-                    if key_code.is_printable() {
-                        //FIXME: this will fail to propogate key combinations such as alt+s
+
```

Comment:
The logic here is tricky.

The problem with alt+s is that we don't actually know what the character value of 's' is, here. For `WM_KEYDOWN` (no alt key pressed) the system will call [`TranslateMessage`](https://docs.microsoft.com/en-ca/windows/win32/api/winuser/nf-winuser-translatemessage) and then we'll get a `WM_CHAR` message with the actual character value; but there's no equivalent for `WM_SYSKEYDOWN`.

How do we get the actual character value? I'm not sure, but I can think of a couple options:

- see if we can call `TranslateMessage` ourselves in this case, and then just handle `WM_CHAR` as normal;
- see if there's another API where we can look up the current char value of this key in the current keyboard layout, and then use that value to construct the appropriate `KeyEvent`.
- or something else entirely?